plotting/latex_tables/defines.tex

```diff
@@ -0,0 +1,259 @@
+%!TEX root =  main.tex
+
+\newcommand{\punt}[1]{}
+\newcommand{\calU}{{\cal U}}
+\newcommand{\calC}{{\cal C}}
+\newcommand{\calB}{{\cal B}}
+\newcommand{\calS}{{\cal S}}
+\newcommand{\calF}{{\cal F}}
+\newcommand{\calD}{{\cal D}}
+
+
+
+
+
+
+%\usepackage{ntheorem}
```

Comment:
same with these package comments. I'm assuming in this case that these are just part of your template and aren't necessary